__init__.py

```diff
@@ -572,8 +572,8 @@ def play_something(self, message):
         if artist:
             LOG.info("\tBut it has to be by " + artist)
             query = 'artist:' + artist
-            res = self.spotify.search(query, type='track')
-            self.play(data=res, data_type='track')
```

Comment:
I think there was actually a reason for this, and it wasn't an oversight. It might be that self.play() wasn't handling the artist type correctly as a data_type? This might require testing just to make sure it does the right thing